ssh/browser/wasm/wasm_js.go

```diff
@@ -0,0 +1,406 @@
+// Copyright (c) 2022 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package that generates JS bindings for Tailscale via WebAssembly.
```

Comment:
// The wasm package is a WebAssembly module that provides Tailscale APIs to JavaScript.

?